packages/viewport/src/listener.native.js

```diff
@@ -26,9 +25,8 @@ const matchWidth = ( operator, breakpoint ) => {
 
 const addDimensionsEventListener = ( breakpoints, operators ) => {
 	const setIsMatching = () => {
-		const matches = reduce(
-			breakpoints,
-			( result, width, name ) => {
+		const matches = Object.entries( breakpoints ).reduce(
+			( result, [ name, width ] ) => {
 				Object.entries( operators ).forEach(
```

Comment:
Here both `operators` and `breakpoints` are also static and `Object.entries` could be lifted up.

Reply:
Done as part of https://github.com/WordPress/gutenberg/pull/44572/commits/7f17ee95db046ca513abc06940480ab34b08f814.